src/main/java/uk/gov/hmcts/ccd/domain/model/definition/RoleToAccessProfileDefinition.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.ccd.domain.model.definition;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RoleToAccessProfileDefinition {
+
+    private String caseTypeId;
+    private Boolean disabled;
+    private Boolean readOnly;
+    private String authorisations;
+    private String accessProfiles;
```

Comment:
I've not yet got to the points of code where these are being used, but maybe we could just have them as `List<String>` here?

Reply:
we have introduced helper methods for these